Functionality_Tests/FunctionalityTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.WaitHelpers;
+using OpenQA.Selenium.DevTools.V115.Page;
+
+namespace Functionality_Tests_Suit
+{
+    [TestFixture]
+
```

Comment:
blank line :) review your code for the rule

Reply:
fixed